crates/gitbutler-command-context/src/lib.rs

```diff
@@ -32,6 +35,10 @@ impl CommandContext {
         &self.git_repo
     }
 
+    pub fn db(&mut self) -> &mut but_db::DbHandle {
```

Comment:
I'd definitely prefer to do this lazily - with `&mut self` that's easy, and won't spawn a database each time `git_head()` is called (this is where it failed for me with "database locked".